packages/pwa-kit-react-sdk/src/ssr/universal/contexts/index.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+
+export const AppContext = React.createContext({})
+
+// THINK: Should we have a state for context so it can be manipulated in the future? If we want to do this,
```

Comment:
I have the same doubt while doing the configuration work.

I feel strongly that this context should be immutable and it should never allow users to update it. Because it's dangerous to be able to mutate this context and cause entire app to re-render.